diff --git a/src/pathnames.h b/src/pathnames.h
index 755dc0c..2a0c440 100644
--- a/src/pathnames.h
+++ b/src/pathnames.h
@@ -30,12 +30,17 @@
 # include <paths.h>
 #endif /*BSD*/
 
+#ifdef __MVS__
+#include "cron-zos-paths.h"
+#else
 #include "cron-paths.h"
+#endif
 
 			/* where should the daemon stick its PID?
 			 * PIDDIR must end in '/'.
 			 * (Don't ask why the default is "/etc/".)
 			 */
+#ifndef __MVS__
 #ifdef CRON_PID_DIR
 # define PIDDIR CRON_PID_DIR "/"
 #else
@@ -48,6 +53,7 @@
 #define PIDFILE		"crond.pid"
 #define _PATH_CRON_PID	PIDDIR PIDFILE
 #define REBOOT_LOCK     PIDDIR "cron.reboot"
+#endif //! def __MVS__
 
 #ifndef _PATH_BSHELL
 # define _PATH_BSHELL "/bin/sh"
